src/scripts.js

```diff
@@ -256,9 +256,38 @@ function pressEnterSearch(event) {
 
 function searchRecipes() {
   showAllRecipes();
-  let searchedRecipes = recipeData.filter(recipe => {
-    return recipe.name.toLowerCase().includes(searchInput.value.toLowerCase());
+  let searchedRecipes = [];
+
+
+
+  recipeData.forEach(recipe => {
```

Comment:
This looks good, Matt! I do think as we work through all of this more, we will be able to do some refactoring as well as move this function to be come a method in the recipe class and call it in scripts.js.